lang/octave/README.md

```diff
@@ -0,0 +1,271 @@
+# GNU Octave
+
+![octave-logo](octave-logo.svg)
+
+[GNU Octave](https://www.gnu.org/software/octave/) is a free and open source
+(FOSS) high-level language for array manipulation and linear algebra (very 
+similar to [MATLAB](https://www.mathworks.com/products/matlab.html)). It ships 
+with an IDE and is very good for things like curve-fitting, modeling and 
+parameter optimization, signal processing, and image manipulation.  
+
+In this example, we'll create an Ubuntu 18.04 based container with a basic
+Octave installation. The definition file will allow for easily starting the 
+Octave IDE, or running a script.
+
+## writing the definition file
+
+First, create a directory for your Octave assets. This step is optional, but it 
+will help keep your space tidy.
+
+```
+$ mkdir ~/octave && cd ~/octave
+```
+
+Open an editor (such as `nano` or `vim`) and copy the following contents into a 
```

Comment:
Where's the `micro` editor?
```
Open an editor (such as: `micro`, `vim`, or `nano`) and copy the following contents into a 
```

Reply:
`micro` is really not a "standard" text editor imho.  It's not even in the Ubuntu repos and I've never heard of it before we started putting it in the `README.md`s.  

afaik, [atom](https://atom.io/) is probably more popular, and we are not including it.  

There are probably dozens of other text editors we could include, but I prefer to stick to the "standards".  I see `vim` as the standard for people who know what they are doing and `nano` as the standard for those who don't.  `emacs` is a standard too, but... ick.  😸